American Whitewater/ViewControllers/Reaches/RunMapViewController.swift

```diff
@@ -168,17 +168,16 @@ class RunMapViewController: UIViewController, MKMapViewDelegate, CLLocationManag
     }
     
     func mapView(_ mapView: MKMapView, didUpdate userLocation: MKUserLocation) {
-        let userLat = userLocation.coordinate.latitude
-        let userLon = userLocation.coordinate.longitude
+        let userCoord = userLocation.coordinate
         
         // check if we need to update locations
-        if abs(userLat - DefaultsManager.latitude) > 0.01 ||
-            abs(userLon - DefaultsManager.longitude) > 0.01 {
+        if userCoord.hasChanged(from: DefaultsManager.shared.coordinate, byMoreThan: 0.01) {
             print("Updating distances of reaches")
+        
+            // FIXME: this doesn't actually update reach distances. Should it?
```

Comment:
Yeah I think it should.

Reply:
👍🏼

How do you feel about leaving this comment as a marker and revisiting later?

I started to make this change. But it's a weird thing to be doing, couldn't we get rid of pre-calculating distances every time the user's location changes, and find within a range of lat/lon instead? And if the pre-calculation really is necessary, it still shouldn't be the run map controller's job to deal with it -- somewhere else we should be watching the saved location change and doing any work that's required.

Reply:
Yep, agree with all that, let's defer here.